drgn/helpers/linux/sched.py

```diff
@@ -209,3 +210,12 @@ def task_since_last_arrival_ns(task: Object) -> int:
     rq_clock = task_rq(task).clock.value_()
 
     return rq_clock - arrival_time
+
+
+def thread_group_leader(t: Object) -> bool:
```

Comment:
The parameter is documented as `task`, which is a better name anyways, so rename `t` -> `task`.